stock_inventory_cost_info/hooks.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+
+def pre_init_hook(cr):
+    cr.execute(
+        """ALTER TABLE stock_inventory_line
+    ADD COLUMN adjustment_cost numeric
+    DEFAULT 0"""
+    )
+
+    cr.execute(
```

Comment:
Maybe an explanation on this as setting a default and then removing it ???

Reply:
Please @ernestotejeda, can you clarify if a default is not really necessary and reduce code?

Reply:
It would not be enough create a column without  the default, I do not see where the field is filled..

Reply:
Creating it without default would lead to null values. We should think if we want such null values and how they are seen in pivot table

Reply:
Oks...